src/test/java/uk/gov/hmcts/ccd/ITSecurityUtils.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.ccd;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.ccd.data.SecurityUtils;
+import uk.gov.hmcts.ccd.security.idam.IdamRepository;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+
+@Service
+@Slf4j
+@Primary
+public class ITSecurityUtils extends SecurityUtils {
```

Comment:
where do we use this?

Reply:
This is used in test scope - by all the integration tests
they used to throw NPE on `getUserToken()` and `getUserRolesHeader()`